### DISCUSSION TITLE
Is it possible to add a query which sets sequence values within a database task?

### DISCUSSION THREAD

**Discussion by LawrenceLau2020** [dbeaver/dbeaver#34383] [Q&A] [answered]
I have an ETL data transfer task in DBeaver where I'm extracting data from a SQL Server db into a Postgres database, but I'd like to run a script basically at the end of the ETL process where it would update the sequence ids for Postgres tables.

`SELECT setval('postgrestable_id_seq', (SELECT COALESCE(MAX(id), 1) FROM postgrestable));`

If I add this as a query, it seems that in the table mapping section wants to map it to a table, but all I want is to just run this query.
Is this possible to do?
Thanks in advance.

**Comment by LawrenceLau2020** [comment]
Just to confirm, f I set the mapping to skip, will it still run the query?

![image](https://github.com/dbeaver/dbeaver/assets/68400651/11a68a3a-124e-4d00-9d12-8d8b82d3edaf)

Thanks!

**Comment by E1izabeth** [comment, marked as answer]
There is a feature called Composite tasks in DBeaver Enterprise and Ultimate versions. I suppose, that's what you need. You can try a trial for free https://dbeaver.com/edition/
Here is a wiki page in this matter https://dbeaver.com/docs/dbeaver/Composite-Tasks/
A video guide https://www.youtube.com/watch?v=G_fdAD57rEY
An article guide https://dbeaver.com/2022/10/07/composite-tasks-and-how-to-create-them/
